lib/datarockets/style/cop/layout/array_alignment_extended.rb

```diff
@@ -64,14 +64,15 @@ def fixed_indentation?
           end
 
           def base_column(node, args)
-            if fixed_indentation?
-              lineno = target_method_lineno(node)
```

Comment:
@lazycoder9 @AleksSenkou  what do you think about it? I'm just not sure that this code is clearer than the old one. 

P.S. I just wanted to move these four lines into the separate method 😅